dubbo-metadata/dubbo-metadata-api/src/main/java/org/apache/dubbo/metadata/report/support/AbstractMetadataReport.java

```diff
@@ -102,11 +105,9 @@
     public AbstractMetadataReport(URL reportServerURL) {
         setUrl(reportServerURL);
         // Start file save timer
-        String defaultFilename = System.getProperty("user.home") +
-                "/.dubbo/dubbo-metadata-" +
-                reportServerURL.getApplication() + "-" +
-                replace(reportServerURL.getAddress(), ":", "-") +
-                ".cache";
+        String defaultFilename = System.getProperty(USER_HOME) + DUBBO_METADATA_KEY +
```

Comment:
According to the context, the name of the constant should not be XX_KEY, right?
DUBBO_METADATA_KEY & CACHE_KEY

Reply:
done